resources/hooks/local/pre-commit

```diff
@@ -8,5 +8,8 @@
 # Fetch the GIT diff and format it as command input:
 DIFF=$(git -c diff.mnemonicprefix=false --no-pager diff -r -p -m -M --full-index --no-color --staged | cat)
 
+# Grumphp env vars
+export GRUMPHP_GIT_WORKING_DIR=$(git rev-parse --show-toplevel)
+
 # Run GrumPHP
 (cd "${HOOK_EXEC_PATH}" && printf "%s\n" "${DIFF}" | $(EXEC_GRUMPHP_COMMAND) $(HOOK_COMMAND) '--skip-success-output')
```

Comment:
If ${GRUMPHP_GIT_WORKING_DIR} would be added  in front of ${HOOK_EXEC_PATH} then grumphp would work on any level not just git root.

Reply:
Thanks for your remark.
Can you add some additional information on what you want this to do exactly?
When running pre-commit, git always takes the git-root as cwd. Or isn't this always the case?

Reply:
I have a case where git root is not the same as composer.json and grumphp.yml. In that case working directory is set to grumphp.yml directory. Testing with setup - [git root] -> [drupal (grumphp.yml and composer.json)]. After install i have lst line : 
`(cd "drupal/" && printf "%s\n" "${DIFF}" | 'exec' './vendor/bin/grumphp' 'git:pre-commit' '--skip-success-output')` That can not recognize files staged.

Reply:
Before fix project/drupal/web only project is working. After all 3 levels are working.

Reply:
Tries this out:

```sh
mkdir tmp
cd tmp
git init
mkdir drupal
cd drupal
composer init
composer config repositories.grumphp-shim vcs https://github.com/veewee/grumphp
echo 'vendor' >> .gitignore
composer require --dev phpro/grumphp:dev-improved-paths
vim grumphp.yml
./vendor/bin/grumphp r
git commit -m"test"
```

The pre-commit looks like you mentioned above and everything seems to work as expected.
Can you tell me the steps on how to get the error message?

(Or is it when you run the git hook directly from the drupal directory? `../.git/hooks/pre-commit`? In that case it seems a better idea to run `./vendor/bin/grumphp git:pre-commit`, ATM we assume the git hook always runs from git root during pre-commit)

Reply:
Ok. After doing a test in clean setup - it seems to work. It might be some weird issue from debugging things. Looks like there are no need for any change. Keep up the good work.